plugins/explore/api-report.md

```diff
@@ -31,14 +30,17 @@ export const catalogEntityRouteRef: ExternalRouteRef<
   true
 >;
 
-// @public (undocumented)
-export const DomainCard: (props: { entity: DomainEntity }) => JSX.Element;
-
 // @public (undocumented)
 export const DomainExplorerContent: (props: {
   title?: string | undefined;
 }) => JSX.Element;
 
+// @public (undocumented)
+export const EntityExplorerContent: (props: {
```

Comment:
`Entity*Content` is a pattern that's used for routable extensions that are intended for the entity pages. I think we'll need to tweak the naming here a little bit. Perhaps `CatalogKindExploreContent`?